Makefile.kokkos

```diff
@@ -327,35 +327,27 @@ KOKKOS_INTERNAL_USE_ARCH_PASCAL60 := $(call kokkos_has_string,$(KOKKOS_ARCH),Pas
 KOKKOS_INTERNAL_USE_ARCH_NVIDIA := $(shell expr $(KOKKOS_INTERNAL_USE_ARCH_KEPLER30)  \
                                               + $(KOKKOS_INTERNAL_USE_ARCH_KEPLER32)  \
                                               + $(KOKKOS_INTERNAL_USE_ARCH_KEPLER35)  \
                                               + $(KOKKOS_INTERNAL_USE_ARCH_KEPLER37)  \
-                                              + $(KOKKOS_INTERNAL_USE_ARCH_PASCAL61)  \
-                                              + $(KOKKOS_INTERNAL_USE_ARCH_PASCAL60)  \
-											  + $(KOKKOS_INTERNAL_USE_ARCH_VOLTA70) \
-											  + $(KOKKOS_INTERNAL_USE_ARCH_VOLTA72) \
-											  + $(KOKKOS_INTERNAL_USE_ARCH_TURING75) \
                                               + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL50) \
                                               + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL52) \
-                                              + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL53))
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL53) \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_PASCAL61)  \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_PASCAL60)  \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_VOLTA70)   \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_VOLTA72)   \
+                                              + $(KOKKOS_INTERNAL_USE_ARCH_TURING75)  \
+					      + $(KOKKOS_INTERNAL_USE_ARCH_AMPERE80))
 
 #SEK: This seems like a bug to me
 ifeq ($(KOKKOS_INTERNAL_USE_ARCH_NVIDIA), 0)
   KOKKOS_INTERNAL_USE_ARCH_MAXWELL50 := $(call kokkos_has_string,$(KOKKOS_ARCH),Maxwell)
   KOKKOS_INTERNAL_USE_ARCH_KEPLER35 := $(call kokkos_has_string,$(KOKKOS_ARCH),Kepler)
-  KOKKOS_INTERNAL_USE_ARCH_NVIDIA := $(shell expr $(KOKKOS_INTERNAL_USE_ARCH_KEPLER30)  \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_KEPLER32)  \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_KEPLER35)  \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_KEPLER37)  \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_PASCAL61)  \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_PASCAL60)  \
-												+ $(KOKKOS_INTERNAL_USE_ARCH_VOLTA70) \
-												+ $(KOKKOS_INTERNAL_USE_ARCH_VOLTA72) \
-												+ $(KOKKOS_INTERNAL_USE_ARCH_TURING75) \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL50) \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL52) \
-                                                + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL53))
+  KOKKOS_INTERNAL_USE_ARCH_NVIDIA := $(shell expr $(KOKKOS_INTERNAL_USE_ARCH_KEPLER35)  \
+                                                + $(KOKKOS_INTERNAL_USE_ARCH_MAXWELL50))
```

Comment:
What the deal with that change?

Reply:
the conditional was redundant. You only enter this if none of the above were set, and the only ones to check for at this pointer were general Maxwell/Kepler which set Maxwell50 and Kepler35 respectively. So you only need to some up those two at this point.